daemon/volumes.go

```diff
@@ -27,9 +28,12 @@ type mounts []container.Mount
 
 // volumeToAPIType converts a volume.Volume to the type used by the Engine API
 func volumeToAPIType(v volume.Volume) *types.Volume {
+	createdAt, _ := v.CreatedAt()
+	createdAtStringFormat := createdAt.Format(time.RFC3339)
 	tv := &types.Volume{
-		Name:   v.Name(),
-		Driver: v.DriverName(),
+		Name:      v.Name(),
+		Driver:    v.DriverName(),
+		CreatedAt: createdAtStringFormat,
```

Comment:
Right now `createdAtStringFormat` is used only here. Thus IMHO it can be removed and this line will look like this:

```golang
CreatedAt: createdAt.Format(time.RFC3339),
```